packages/babel-preset-mc-app/index.js

```diff
@@ -68,8 +68,6 @@ module.exports = function getBabePresetConfigForMcApp(api, opts = {}) {
           // This is probably a fine default to help trim down bundles when
           // end-users inevitably import '@babel/polyfill'.
           useBuiltIns: !options.disableCoreJs ? 'entry' : false,
-          // Do not transform modules to CJS
-          modules: false,
```

Comment:
Otherwise loading `.js` files with `import` statements fail.

I double checked with the CRA babel preset (since we base our preset from theirs) and they have also removed it since a couple of years: https://github.com/facebook/create-react-app/blob/a179b05300c24dd54349c28a94005999492c2567/packages/babel-preset-react-app/create.js#L78-L89